TrafficCapture/dockerSolution/src/main/docker/migrationConsole/Dockerfile

```diff
@@ -40,14 +40,21 @@ COPY lib /root/lib
 WORKDIR /root/lib/console_link
 RUN pipenv install --system --deploy --ignore-pipfile
 
-# Experimental console API, not started by default
+# Experimental console API, uncomment to start.
+# ENV ENABLE_CONSOLE_API=true
 COPY console_api /root/console_api
 WORKDIR /root/console_api
 RUN pipenv install --system --deploy --ignore-pipfile
+
 WORKDIR /root
-#CMD pipenv run python manage.py runserver_plus 0.0.0.0:8000
 
 # Ensure bash completion is installed
 RUN apt-get install -y bash-completion
 
-CMD /root/loadServicesFromParameterStore.sh && tail -f /dev/null
+ENV ENABLE_CONSOLE_API=${ENABLE_CONSOLE_API:-false}
+CMD if [ "$ENABLE_CONSOLE_API" = "true" ]; then \
```

Comment:
Not a blocker for me but I personally find it cleaner to just have a default command here and then overwrite the command as needed,  to limit environment variables and conditional logic in the Dockerfile. It seems like we are not using this in CDK and just overwriting the command

Reply:
Updated